tests/data/loaders/test_load_events.py

```diff
@@ -26,6 +26,15 @@ def __init__(self):
             self.ev_time = "origTime"
             self.ev_value = "origValue"
 
+        # Aid setting a list of events (like that from config) but getting a dict like ConfigProvider returns
+        @property
+        def events(self):
+            return self._events
+
+        @events.setter
+        def events(self, event_list):
```

Comment:
can we add a type annotation for event_list?

Reply:
Done